### PR TITLE
fix:added the email validator

### PR DESCRIPTION
**Pull request by avishmehta68710**

### CODE REVIEW

**Comment by avishmehta68710** [comment]
can you tell me why my pr checks are failing this time i did everything which you mentioned earlier

**Comment by arteevraina** [comment]
> can you tell me why my pr checks are failing this time i did everything which you mentioned earlier

@avishmehta68710 seems like some formatting issues with the code. But, still I think we can merge this PR after slight changes.

File: lib/constants/validators.dart [hunk 1]
```diff
@@ -54,3 +54,18 @@ Function(String) bloodGroupValidator = (String bloodGroup) {
 
   return null;
 };
+
+Function(String) emailValidator = (String emailaddress) {
+  String email = r"^[a-zA-Z0-9.a-zA-Z0-9.!#$%&'*+-/=?^_`{|}~]+@[a-zA-Z0-9]+\.[a-zA-Z]+";
+  bool isValid = RegExp(email).hasMatch(emailaddress);
```
Comment (arteevraina) [inline lib/constants/validators.dart:60]: rename this variable as `emailAddress`. Follow camelCasing for naming variables :)

**Review by arteevraina — CHANGES_REQUESTED** [review]

**Comment by arteevraina** [comment]
Also it is a good practice to write what the code does in PR description.

**Comment by avishmehta68710** [comment]
Hi 
Please check now i have applied the changes but still getting this error

**Review by arteevraina — APPROVED** [review]

**Comment by arteevraina** [comment]
@avishmehta68710 LGTM!

**Pull request merged**